src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Shuffle.txt

```diff
@@ -0,0 +1,5 @@
+>> CountRows(Shuffle([-2, -1, 5, 1, 2]))
+5
+
+>> Sort(Shuffle([-2, -1, 5, 1, 2]), Value)
+[-2,-1,1,2,5]
```

Comment:
Can you add a few other test cases:
(I looked at our existing tests for the JS implementation for inpsiration)

Arrays that contain:
- duplicate values 
- errors, blanks,
- non numbers

I think these will all just work with the current implementation. 


Also: 
Shuffle(1/0)  // pass in an error instead of array
Shuffle(blank()) // pass in blank instead of array 

Shuffle([]) //  empty array

Reply:
I added the unit tests